using-mobile-extensions/mobile-core/rules-engine/README.md

```diff
@@ -45,6 +45,12 @@ At the start of a new application session that includes the Experience Platform
 
 **Tip**: Conditions can be one match or a group of matches.
 
+### **Re-evaluating events**
+
+Since downloading rules will spend some time, serveral incoming events occured at this time will not be evaluated by the Rules Engine. So the Experience Platform Mobile SDK will cache the first several imcoming events until the rules get downloaded and re-evaluate those cached events with the downloaded rules.
```

Comment:
Add a line tell on which version the change was added

Reply:
On the old versions of Experience Platform Mobile SDKs( prior to iOS version 1.6.2, Android version 1.5.4), after the first launch of the app, it always takes some time to download the rules from the remote servers, so Rules Engine won't be able to evaluate the first several events until the rules are loaded. Start from iOS version 1.6.2 and Android version 1.5.4, we add the capability to cache the events before the Launch UI rules is downloaded and will re-evaluate those events afterward. This change is mainly to enable the trigger of Posback based on the install event.

Reply:
Little bit of update from Jiabin's notes:
On older versions of Experience Platform Mobile SDKs ( prior to iOS version 1.6.2, Android version 1.5.4), after the first launch of the app, it always takes some time to download the rules from the remote servers. During this time, Rules Engine won't be able to evaluate the first several events until the rules are loaded. Starting from iOS version 1.6.2 and Android version 1.5.4, we add the capability to cache the events before the rules are downloaded and will re-evaluate those events afterward. This change is mainly to enable the trigger of Posback based on the install event.